Dart/src/com/jetbrains/lang/dart/analyzer/DartAnalysisServerService.java

```diff
@@ -169,6 +170,14 @@ public void computedNavigation(String file, List<NavigationRegion> regions) {
     }
 
     @Override
+    public void computedOverrides(String file, List<OverrideMember> overrides) {
+      synchronized (myOverrideData) {
+        myOverrideData.put(file, overrides);
+      }
+      forceFileAnnotation(file);
```

Comment:
Notifications often come quickly one after another.
So, we could queue restart requests for unique files and perform them with some delay, like 25 ms.